les/benchmark.go

```diff
@@ -135,7 +135,7 @@ func (b *benchmarkHelperTrie) init(pm *ProtocolManager, count int) error {
 		b.sectionCount, b.headNum, _ = pm.server.bloomTrieIndexer.Sections()
 	} else {
 		b.sectionCount, _, _ = pm.server.chtIndexer.Sections()
-		b.sectionCount /= (params.CHTFrequencyClient / params.CHTFrequencyServer)
+		b.sectionCount /= params.CHTFrequencyClient / params.CHTFrequencyServer
```

Comment:
Whilst the changes are generally ok, could be revert these specific type of changes where `var1 op= var2 op var3`. Removing the parentheses only makes the code harder to read.